configuration.json

```diff
@@ -0,0 +1,52 @@
+{

```

Comment:
```
{'reports': 
    {'report_keyword_overview' : {....}, 
     'report_organic_results': {....}, 
      'report_indexed_pages': {....} 
    },
   'final_table_schema': {.....}}
```

Reply:
made two main entities: reports and data_requirements